visualizations/frontend/filters/v2/TDatePicker.vue

```diff
@@ -27,18 +39,21 @@
         </button>
       </div>
 
+      <!-- To hide default clear icon -->
```

Comment:
Is that comment accurate? I

Reply:
Yes I am using `<div slot="icon-clear"></div>` to hide the default icon of this component, it does not match with our design.